### PR TITLE
feat(bot-utils): use price-api as first price source

### PR DESCRIPTION
**Pull request by Louis-Amas**

### CODE REVIEW

File: packages/bot-utils/src/util/priceUtils.ts [hunk 1]
```diff
@@ -5,6 +5,27 @@ import { fetchJson } from "ethers/lib/utils";
...
+  volume: number;
+  date: string;
+};
+
+/* A lot of CEX does not have pair with USDC, we can convert to usd as the ratio is almost 1:1
+ * It's not critical in this application as if USDC depeg, we will over estimate the actual gas
+ * cost of an arbitrage, and that's ok.
+ **/
+export const converter = (token: string) => {
+  if (token.toUpperCase() === "USDC") {
```
Comment (peterMangrove) [inline packages/bot-utils/src/util/priceUtils.ts:22]: What about USDT?

**Review by peterMangrove — CHANGES_REQUESTED** [review]

File: packages/bot-utils/src/util/priceUtils.ts [hunk 2]
```diff
@@ -29,22 +57,28 @@ export class PriceUtils {
     getJson: () => Promise<any>;
     price: () => Promise<Big | undefined>;
   } {
-    const cryptoCompareUrl = `https://min-api.cryptocompare.com/data/price?fsym=${inToken}&tsyms=${outToken}`;
+    const priceApiUrl = `https://prod-price-api-32327e5e7126.herokuapp.com/prices/${converter(
```
[the remark this answers is not in the document]
Comment (Louis-Amas) [reply packages/bot-utils/src/util/priceUtils.ts:60]: Ok I will add that

File: packages/bot-utils/src/util/priceUtils.ts (hunk 1 above)
Comment (Louis-Amas) [reply packages/bot-utils/src/util/priceUtils.ts:22]: USDT is supported on CEXes but we could add that as a configuration

Comment (peterMangrove) [reply packages/bot-utils/src/util/priceUtils.ts:22]: Thats actually a good idea. Maybe have a configuration, where you can map from one token/symbol to another, this way it is really easy to add new tokens that are not supported by CEXes

**Comment by Louis-Amas** [comment]
Closing this PR as I will include the change in the new arb bot with market order

**Pull request closed without merging**